pyodide-build/pyodide_build/cli/new_package.py

```diff
@@ -0,0 +1,87 @@
+from pathlib import Path
```

Comment:
Same here the file name could be `new_recipe.py` or something similar. That's what conda call these .yaml files.